src/components/NoteDisplay/NonDeleteMode.js

```diff
@@ -0,0 +1,49 @@
+import React, { useContext } from "react";
+import { useHistory } from 'react-router-dom';
+import Icon from "../shared/Icon";
+import Button from "../shared/Button";
+import iconDir from "../icon.svg";
+import globalTypes from "../../GlobalTypes";
+import { AppContext } from "../../contexts/AppContext";
+import { ThemeContext } from "../../contexts/ThemeContext";
+
+export default function NonDeleteMode({ noteId }) {
+    const { state, dispatch } = useContext(AppContext);
+    const { setDarkMode, darkMode } = useContext(ThemeContext);
+    const history = useHistory();
+
+    return (
+        <>
+            <Button onClick={() => dispatch({type:globalTypes.toggleSidebar})}>
```

Comment:
You can relocate the function above the return statement with the name of `toggleSidebar` so then pass it to button like this `<Button onClick={toggleSidebar}`